docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+
+services:
+  db:
+    image: ankane/pgvector:latest
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgres
+    volumes:
+      - /var/lib/postgresql/data
+    network_mode: host
```

Comment:
**Question:** Do we not need to explicitly expose ports because we use host networking?

Reply:
Right, exposing ports doesn't work with host networking